mne/source_estimate.py

```diff
@@ -1380,11 +1380,11 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
             How to print info about the time instant visualized.
         smoothing_steps : int
             The amount of smoothing.
-        fmin : float
+        fmin : float | None (Deprecated)
             The minimum value to display.
-        fmid : float
+        fmid : float | None (Deprecated)
             The middle value on the colormap.
-        fmax : float
+        fmax : float | None (Deprecated)
```

Comment:
I think we normally remove the doc for deprecated parameters. @agramfort @Eric89GXL what's your intuition on this?